CUJExecutor/executeCuj

```diff
@@ -64,7 +64,7 @@ then
 	CUJSTART=$(adb logcat -s "clip_start" -v raw -d | sed -n -e '/^1/p')
 	CUJEND=$(adb logcat -s "clip_end" -v raw -d | sed -n -e '/^1/p')
 	CUTAT=$((CUJSTART - RECSTART - 1000)) 
-	LENGTH=$((CUJEND - RECSTART + 1000)) 
+	LENGTH=$((CUJEND - RECSTART + 200)) 
```

Comment:
Why the hardcoded times?

Reply:
Hard coded buffer between start/end of actions and start/end of video.

Reply:
Need to add comments for them.